rust-toolchain.toml

```diff
@@ -1,2 +1,2 @@
 [toolchain]
-channel = "nightly"
+channel = "stable"
```

Comment:
Note that this was only required to make rustfmt use nightly. Nothing in git-cliff seems to require the nightly compiler to work afaik (I may be wrong there)

Reply:
Yup, it was only for formatting